src/CMakeLists.txt

```diff
@@ -180,6 +182,7 @@ SET( brewtarget_UIS
     ${UIDIR}/mashStepEditor.ui
     ${UIDIR}/miscEditor.ui
     ${UIDIR}/fermentableEditor.ui
+    ${UIDIR}/equipmentEditor.ui
```

Comment:
So, my effort to murder equipmentEditor.ui was in vain, and it is risen from the grave?